.github/workflows/go.yml

```diff
@@ -18,6 +18,7 @@ jobs:
         proxysql-image:
           - percona/proxysql:1.2.1
           - perconalab/proxysql:1.3.6
+#          - percona/proxysql2:2.7
```

Comment:
Is this leftover?

Reply:
No, it's plan for future. I tried to make it work, but couldn't.